### PR TITLE
Fix keyboard accessibility

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

File: src/NotificationsApp.vue [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-			tabindex="0"
-			role="button"
-			:aria-label="t('notifications', 'Notifications')"
-			aria-haspopup="true"
-			aria-controls="notification-container"
-			aria-expanded="false"
-			@click="requestWebNotificationPermissions"
-			@keydown.enter="requestWebNotificationPermissions">
+	<HeaderMenu
+		v-if="!shutdown"
```
Comment (nickvergessen) [inline src/NotificationsApp.vue:3]: Some eslints here

**Comment by nickvergessen** [comment]
> HeaderMenu is taken from core/src/components and should probably be moved to nextcloud-vue

Sounds like it.

**Comment by CarlSchwan** [comment]
> > HeaderMenu is taken from core/src/components and should probably be moved to nextcloud-vue
> 
> Sounds like it.

I created https://github.com/nextcloud/nextcloud-vue/pull/2886 but don't think this should be blocking for this PR (we can adapt to the newer component in another PR)

**Comment by nickvergessen** [comment]
Am I doing something wrong?

Neither with notifcation nor without the content is shown to me

#### Without
Bildschirmfoto vom 2022-07-26 10-44-01

### With notification
Bildschirmfoto vom 2022-07-26 10-44-18

**Comment by CarlSchwan** [comment]
> Am I doing something wrong?
> 
> Neither with notifcation nor without the content is shown to me
> #### Without
> 
> Bildschirmfoto vom 2022-07-26 10-44-01
> ### With notification
> 
> Bildschirmfoto vom 2022-07-26 10-44-18

fixed now

File: src/NotificationsApp.vue [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-			role="button"
-			:aria-label="t('notifications', 'Notifications')"
-			aria-haspopup="true"
-			aria-controls="notification-container"
-			aria-expanded="false"
-			@click="requestWebNotificationPermissions"
-			@keydown.enter="requestWebNotificationPermissions">
+	<HeaderMenu v-if="!shutdown"
+		id="notifications"
+		class="Notifications"
```
Comment (nickvergessen) [inline src/NotificationsApp.vue:4]: The dot is not red anymore if there is a new notification.
```suggestion
		class="notifications-button"
```

But then positions are off, so maybe most of the rules directly in `notifications-button` should be removed?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**